src/Behat/Behat/Output/Node/EventListener/JUnit/JUnitOutlineStoreListener.php

```diff
@@ -53,7 +54,18 @@ public function __construct(SuitePrinter $suitePrinter)
      */
     public function listenEvent(Formatter $formatter, Event $event, $eventName)
     {
-        $this->captureOutlineOnBeforeOutlineEvent($event);
+      $this->captureOutlineOnBeforeOutlineEvent($event);
+      if (method_exists($event, 'getSpecificationIterator')) {
+        $class = 'Behat\Gherkin\Node\FeatureNode';
+        foreach ($event->getSpecificationIterator() as $iterator) {
```

Comment:
I would name the variable `$iterator` as `$node` instead as this iterator return `Node`s.